vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -422,6 +419,36 @@ public VimeoAccount authorizeWithClientCredentialsGrantSync() {
         return call;
     }
 
+    /**
+     * Synchronous version of {@link #singleSignOnTokenExchange(String, AuthCallback)}. This is useful
+     * when already running in a background thread, such as when using the Android AccountManager.
+     *
+     * @param token           the authenticated user access token from application A. This <b>cannot</b> be a client
```

Comment:
Does this mean that the token must be a password-grant token? If so, consider saying so in the docs (rather than saying what it can't be), and maybe even capturing this in the method name or argument name.